### PR TITLE
Update: TreePicker shorten by 100px

### PR DESCRIPTION
**Pull request by camjc**
- Lets the TreePicker modal show on projectors without scrolling.
https://trello.com/c/43vgVwjk/4976-update-treepicker-shorten-by-100px-so-that-it-fits-on-a-projector

### CODE REVIEW

File: src/styles/adslotUi/TreePickerSelected.scss [hunk 1]
```diff
@@ -11,15 +11,15 @@
   }
 
   &-scrollable {
-    $scrollable-max-height: 472px + $header-height;
+    $scrollable-height: $treepicker-height + $header-height - 128px;
```
Comment (antklim) [inline src/styles/adslotUi/TreePickerSelected.scss]: Can we create a variable for `128px`?

Comment (camjc) [inline src/styles/adslotUi/TreePickerSelected.scss]: I would if it had some sort of semantic meaning. Its a bit of a magic number to make it look right I'm afraid. I'll try to keep whittling away at that in subsequent prs.

**Comment by antklim** [comment]
LGTM :ideograph_advantage:

File: src/styles/adslotUi/TreePickerSelected.scss (hunk 1 above)
Comment (antklim) [inline src/styles/adslotUi/TreePickerSelected.scss]: Dmitry's style: magicKrakenVariable = 128px :space_invader:

Comment (camjc) [inline src/styles/adslotUi/TreePickerSelected.scss]: haha. :space_invader:

**Comment by etaoins** [comment]
:rocket:

**Pull request merged**